examples/mysql/docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
       MYSQL_DATABASE: superset
       MYSQL_USER: superset
       MYSQL_PASSWORD: superset
+    ports:
+      - "3306:3306"
```

Comment:
It is not necessary to publish the ports because the container will be running on the same network as the app.